src/components/BalanceDetailsTable.tsx

```diff
@@ -88,24 +66,26 @@ const createBalancesDetailViewRow = (
   };
 
   return [
-    <Label key={balance.id}>
-      <Icon uuid={balance.uuid as TUuid} size={'2rem'} />
-      {balance.name}
-    </Label>,
-    <Label key={balance.id}>
-      {balance.accounts &&
-        (balance.accounts.length > 1
-          ? translateRaw('WALLET_BREAKDOWN_DETAIL_ACCOUNTS', {
-              $numOfAccounts: balance.accounts.length.toString()
-            })
-          : translateRaw('WALLET_BREAKDOWN_DETAIL_ACCOUNT', {
-              $numOfAccounts: balance.accounts.length.toString()
-            }))}
-    </Label>,
-    <RowAlignment data-balance={balance.amount} key={balance.id} align="right">
+    <Box variant="rowAlign" data-sortkey={balance.name} key={balance.id}>
+      <AssetIcon mr={'0.5ch'} uuid={balance.uuid as TUuid} size="1.5em" />
```

Comment:
![image](https://user-images.githubusercontent.com/1561200/109646434-7aa67580-7b58-11eb-869e-505b33a149de.png)
This one has issues with some icons

Reply:
This issue is still present

Reply:
Done